src/main/resources/alerts/en.yml

```diff
@@ -0,0 +1,76 @@
+#
+# Refer to the Bukkit API javadoc to find events & properties to use:
+#  https://hub.spigotmc.org/javadocs/bukkit
+# Helpful resources for learning how to use SpEL:
+#  https://docs.spring.io/spring/docs/4.2.x/spring-framework-reference/html/expressions.html
+#  https://dzone.com/articles/learn-spring-expression-language-with-examples
+#
+# You can use the following placeholders:
+#  {tps} - server TPS
+#  {time} - formatted time
+#  {date} - formatted date
+#  {name} - if alert is for a player event, the name of the player
+#  {ping} - if alert is for a player event, the ping of the player
+#  {username} - if alert is for a player event, the username of the player
+#  {displayname} - if alert is for a player event, the display name of the player
+#  {world} - if alert is for a player event, the world the player is in
+#  {embedavatarurl} - if alert is for a player event, the avatar url for the player's head, otherwise, the bot's avatar url
+#  {botavatarurl} - the bot's avatar url
+#  {botname} - the bot's name
+#
+# You can also use SpEL expressions with ${expression...}, such as...
+#  - Getting a player's IP: ${player.address.address.hostAddress}
+#  - Getting a player's game mode: ${player.gameMode.name()}
+#  - Getting a player's linked Discord account ID: ${discordsrv.accountLinkManager.getDiscordId(player.uniqueId)}
+#  - Getting the online player count: ${server.onlinePlayers.size()}
+#  - Getting the status of DiscordSRV's connection to Discord: ${jda.status.name()}
+#  - Checking if a player is in a specific world: ${player.world.name == 'world_the_end'}
+#  - Checking if it's day in the world that the player is in: ${player.world.time > 0 && player.world.time < 13000}
+# You get the point.
+#
+# Syntax/defaults:
+#  - Trigger: <event name or /command>
+#    Conditions:
+#      - property == < > <= >= value etc
+#    Channel: <discordsrv channel name>
+#    IgnoreCancelled: true # only for event alerts
+#    Content: ""
+#    Webhook:
+#      Enable: false
+#      AvatarUrl: "{botavatarurl}"
+#      Name: "{botname}"
+#    Embed:
+#      Enabled: true
+#      Color: "#00ff00"
+#      Author:
+#        ImageUrl: "{embedavatarurl}"
+#        Name: "{username} did... something."
+#        Url: ""
+#      ThumbnailUrl: ""
+#      Title:
+#        Text: ""
+#        Url: ""
+#      Description: ""
+#      Fields: []
+#      ImageUrl: ""
+#      Footer:
+#        Text: ""
+#        IconUrl: ""
+#      Timestamp: false
+#
+# Example config to send messages to the "fish" DiscordSRV channel when a player successfully catches a fish
+# Alerts:
+#   - Trigger: PlayerFishEvent
+#     Channel: fish
+#     Conditions:
+#       - state.name() == 'CAUGHT_FISH'
+#     Embed:
+#       Color: "#00ff00"
+#       Author:
+#         ImageUrl: "{embedavatarurl}"
+#         Name: "{name} caught a ${caught.itemStack.type.name()}!"
+#
```

Comment:
Should note that variables here: https://github.com/DiscordSRV/DiscordSRV/pull/759/files#diff-7a39fe42227c6bd8e51118b8b242ce13R122-R127 are available somewhere